src/components/Links/ButtonLink.tsx

```diff
@@ -2,16 +2,32 @@ import classNames from 'classnames';
 import { Link } from 'gatsby';
 import React from 'react';
 
-type Props = { to: string; className?: string; newWindow?: boolean };
+type ButtonSizes = 'sm' | 'md' | 'lg';
+
+type Props = {
+  to: string;
+  className?: string;
+  newWindow?: boolean;
+  size?: ButtonSizes;
+  outline?: boolean;
+};
 
 // TODO: Refactor all those link components to share more logic
 export const ButtonLink: React.FC<Props> = ({
   to,
   className = '',
   newWindow = false,
+  outline = false,
+  size = 'md',
```

Comment:
I feel like true/false props would work better here. `<ButtonLink big outline to…>`.

You could create Props that make sure that only one (big or small or none) is selected. However, that's a bit much IMO.